src/main/java/gov/usgs/wma/waterdata/DbConfig.java

```diff
@@ -0,0 +1,52 @@
+package gov.usgs.wma.waterdata;
+
+import javax.sql.DataSource;
+
+import org.springframework.boot.autoconfigure.jdbc.DataSourceProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@Configuration
+public class DbConfig {
+
+	@Bean
+	@ConfigurationProperties(prefix="spring.datasource-transform")
+	@Primary
+	@Profile("default")
```

Comment:
Is the @Profile needed here or line 26? Might be an unecessary artifact from where this was based on.

Reply:
I removed it.